docs/application/dotnet/guides/connectivity/internet-download.md

```diff
@@ -9,6 +9,10 @@ You can use the following Internet and content download features in your .NET ap
 
     You can get information about the connected network, and set up and configure network and socket connections. You can also track connection information, such as the size of the sent and received data.
 
+-   [WebView](webview.md)
+
+    You can access Web pages and Web content in your application. You can also use various features for Web browsing, such as loading and displaying Web pages and navigating through the browsing history.
```

Comment:
The Web View feature enables you to access web pages and other web content in your application. In addition, it also provides various other features for web browsing, such as loading, displaying web pages and navigating through the browsing history.

Reply:
Done.

Reply:
Change  Web View --> WebView

Reply:
Done.